src/test/java/org/xbill/DNS/ResolverConfigTest.java

```diff
@@ -0,0 +1,88 @@
+package org.xbill.DNS;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.DisabledOnOs;
+import org.junit.jupiter.api.condition.EnabledOnOs;
+import org.junit.jupiter.api.condition.OS;
+
+import java.net.URISyntaxException;
+import java.nio.file.Paths;
+import java.util.Arrays;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.assumeFalse;
+
+class ResolverConfigTest {
+
+	@AfterEach
+	void tearDown() {
+		ResolverConfig.refresh();
+	}
+
+	@Test
+	void findProperty_Success() {
+		String[] dnsServers = { "server1", "server2" };
+		// intentionally adding duplicate search entries for testing
+		String[] dnsSearch = { "dnsjava.org", "example.com",
+			"dnsjava.org" };
+		Name[] searchPath = Arrays.stream(dnsSearch)
+			.map(s -> Name.fromConstantString(s, Name.root))
```

Comment:
If this call is the only reason you added Name.fromConstantString(String, Name), why don't you simply put the trailing dot for the root to the string array?

Reply:
Future tests may look at the inputs from other sources for search path (ipconfig output), and those sources usually don't have the trailing dot.  We could adjust the inputs, but I feel conflicted relying on massaging test resources to have trailing dots.  That being said, it's your call.  I can go either way.

Reply:
The overload is anyway not necessary, this is equivalent: `Name[] searchPath = Arrays.stream(dnsSearch).map(s -> Name.fromConstantString(s + "."))`

Reply:
Updated and reverted the Name addition